CodeEditModules/Modules/GitClient/src/Models/Commit.swift

```diff
@@ -23,18 +23,29 @@ public struct Commit: Equatable, Hashable, Identifiable {
     public var commitBaseURL: URL? {
         if let remoteURL = remoteURL {
             if remoteURL.absoluteString.contains("github") {
-                return remoteURL.deletingPathExtension().appendingPathComponent("commit")
+                return parsedRemoteUrl(domain: "https://github.com", remote: remoteURL)
             }
             if remoteURL.absoluteString.contains("bitbucket") {
-                return remoteURL.deletingPathExtension().appendingPathComponent("commits")
+                return parsedRemoteUrl(domain: "https://bitbucket.org", remote: remoteURL)
             }
             if remoteURL.absoluteString.contains("gitlab") {
-                return remoteURL.deletingPathExtension().appendingPathComponent("commits")
+                return parsedRemoteUrl(domain: "https://gitlab.com", remote: remoteURL)
             }
             // TODO: Implement other git clients other than github, bitbucket here
         }
         return nil
     }
+    func parsedRemoteUrl(domain: String, remote: URL) -> URL {
```

Comment:
make this `private` and add a new line in between row 37 & 38

Reply:
Done